tests/integration/commands/local/lib/test_sam_api_provider.py

```diff
@@ -0,0 +1,195 @@
+import logging
+from unittest import TestCase
+
+from samcli.commands.local.lib.provider import Api
+from samcli.commands.local.lib.sam_api_provider import SamApiProvider
+
+logging.basicConfig(level=logging.INFO)
+
+
+class TestSamStageValues(TestCase):
```

Comment:
There should be in the unit test bucket. Integration tests are for testing as if a customer was running the command (shelling out to `sam local start-api` in this example)